docs/dptp-triage-sop/etcd-slowness.md

```
@@ -10,9 +10,68 @@ these messages are packed together:
+unset ETCDCTL_ENDPOINTS
+REVISION=$(etcdctl endpoint status --write-out fields | sed -nE 's,"Revision" : ([0-9]+),\1,p')
+etcdctl compact $REVISION # needed only a single time on any member
+etcdctl defrag $REVISION --command-timeout=120s 
+```
+
+**Important:**
+
+These commands can take a while depending on the state of the cluster, in any case you can increase
+the value of `--command-timeout` to `300` or even higher if you receive `context deadline exceeded`.
+
+### All-in-one script
```

markdownlint-cli2 (0.23.0)

[warning] Heading style
Expected: setext; Actual: atx

(MD003, heading-style)